gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidNetworkShifter.java

```diff
@@ -97,7 +97,7 @@ public void shiftNetwork(double shiftValue, Network network, TTimestampWrapper t
         try {
             networkShifter.shiftNetwork(shiftValue, network);
         } catch (GlskLimitationException | ShiftingException e) {
-            throw new CseValidShiftFailureException(e.getMessage());
+            throw new CseValidShiftFailureException(e.getMessage(), e.getCause());
```

Comment:
Instead of passing the message and the cause in arguments, why don't you directly pass the original exception?